hack/templates/aws.managed.openshift.io_v1alpha1_ccs_accountclaim_cr_invalid.tmpl

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
```

Comment:
Care to leave a comment at the top of this file indicating how it's an invalid accountclaim?

Reply:
I would just go ahead and remove this file and associated make targets again, as i don't really see the need for this kind of integration test now that we have the unit test properly set up.

Reply:
Sounds good :+1: